maisim/maisim.Game/Screen/MainMenuScreen.cs

```diff
@@ -15,68 +16,103 @@ namespace maisim.Game.Screen
     /// </summary>
     public class MainMenuScreen : osu.Framework.Screens.Screen
     {
+        private Sprite maisimLogo;
+        private MainMenuButton playButton;
+        private MainMenuButton editButton;
+        private MainMenuButton browseButton;
+        private MainMenuButton exitButton;
+
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(TextureStore textureStore)
         {
             InternalChildren = new Drawable[]
             {
                 new Container
                 {
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
+                    Anchor = Anchor.BottomRight,
+                    Origin = Anchor.BottomRight,
                     Size = new Vector2(400, 400),
-                    RelativeSizeAxes = Axes.X,
+                    Position = new Vector2(-70, 0),
                     Children = new Drawable[]
                     {
-                        new Box
-                        {
-                            Anchor = Anchor.Centre,
-                            Origin = Anchor.Centre,
-                            RelativePositionAxes = Axes.Both,
-                            RelativeSizeAxes = Axes.X,
-                            Size = new Vector2(300, 300),
-                            Colour = Color4Extensions.FromHex("f5be39"),
-                        },
                         new FillFlowContainer
                         {
                             Anchor = Anchor.Centre,
                             Origin = Anchor.Centre,
-                            RelativeSizeAxes = Axes.X,
+                            RelativeSizeAxes = Axes.Y,
                             RelativePositionAxes = Axes.Y,
+                            Spacing = new Vector2(0, 10),
                             Children = new Drawable[]
                             {
-                                new MainMenuButton("Play",FontAwesome.Solid.Play)
+                                playButton = new MainMenuButton("Play",FontAwesome.Solid.Play,Color4Extensions.FromHex("73E99B"))
                                 {
                                     Anchor = Anchor.Centre,
                                     Origin = Anchor.Centre,
-                                    Size = new Vector2(300,300),
+                                    Size = new Vector2(400, 60),
+                                    Scale = new Vector2(0),
                                     Action = () => this.Push(new SongSelectionScreen())

```

Comment:
Keep in mind we may want to preload asynchronously the song select screen in the future to prevent any freezes if the user were to have bazillions of beatmaps upon entering.